h2o-algos/src/main/java/hex/tree/FriedmanPopescusH.java

```diff
@@ -143,6 +144,9 @@ static class FindFValue extends MRTask<FindFValue> {
             for (int iRow = 0; iRow < cs[0].len(); iRow++) {
                 for (int k = 0; k < valueToFindFValueFor.length; k++) {
                     int id = ArrayUtils.find(currFValuesNames, currNames[k]);
+                    if (Double.isNaN(valueToFindFValueFor[k]) && Double.isNaN(cs[id].atd(iRow))){
+                        count++;
+                    }
```

Comment:
This is a critical part of missing values implementation.

Reply:
When I am looking for NaN and found NaN then I report success. It make sense.. do they suggest somewhere how to handle missing values? Maybe to share the source here for future questions.